common/sys/big5_gen.py

```diff
@@ -12,37 +13,37 @@
 # b2u
 sys.stderr.write('Generating B2U data...\n')
 b2u = B2U_FILE.readlines()
-b2u = [line.strip().split(' ')
+b2u = [line.strip().split(b' ')
        for line in b2u
-       if line.strip().startswith('0x')]
+       if line.strip().startswith(b'0x')]
 b2u = dict((int(b, 0), int(u, 0)) for (b, u) in b2u)
 
-print """#include <stdint.h>
+print ("""#include <stdint.h>
```

Comment:
Can you remove the space between 'print' and '(' ? Looks more like a function call.